consensus/istanbul/backend/handler.go

```diff
@@ -166,6 +181,11 @@ func (sb *Backend) handleFwdMsg(peer consensus.Peer, payload []byte) error {
 	return nil
 }
 
+// SubscribeNewDelegateSignEvent subscribes a channel to any new delegate sign messages
+func (sb *Backend) SubscribeNewDelegateSignEvent(ch chan<- istanbul.MessageEvent) event.Subscription {
+	return sb.delegateSignFeed.Subscribe(ch)
```

Comment:
I think we should also use an `event.SubscriptionScope` similar to here: https://github.com/celo-org/celo-blockchain/blob/master/core/tx_pool.go#L495

so that we can unsubscribe channels https://github.com/celo-org/celo-blockchain/blob/master/core/tx_pool.go#L480 when we close here: https://github.com/celo-org/celo-blockchain/blob/master/consensus/istanbul/backend/backend.go#L209